wisdem/drivetrainse/gearbox.py

```diff
@@ -127,6 +129,8 @@ def setup(self):
         self.add_input("gearbox_mass_user", val=0.0, units="kg")
         self.add_input("gearbox_radius_user", val=0.0, units="m")
         self.add_input("gearbox_length_user", val=0.0, units="m")
+        self.add_input("gearbox_torque_density", self.options["gearbox_torque_density"], 
```

Comment:
See comment below.  Maybe gearbox_torque_density belongs in geometry yaml so this isn't an awkward handoff.

Reply:
this is now fixed